Framework/API/inc/MantidAPI/DetectorInfo.h

```diff
@@ -5,6 +5,7 @@
 #include "MantidKernel/DateAndTime.h"
 #include "MantidKernel/Quat.h"
 #include "MantidKernel/V3D.h"
+#include "MantidGeometry/Instrument.h"
```

Comment:
Based on the new method signature, this new include is not necessary - however, we should think of a different way of organising the code based on Simon's comments below.